app/services/cms/providers/strapi/mocks/image.rb

```diff
@@ -21,7 +21,7 @@ def self.generate_data
             {
               name: filename,
               alternativeText: Faker::Lorem.sentence,
-              caption: Faker::Lorem.sentence,
+              caption: "",
```

Comment:
Can we add caption as a keyword param on generate_data, and make it nil default rather than an empty string